### ISSUE TITLE
关于模型优化的事情

### ISSUE THREAD

**Issue by Human9000** [Garliccurry/ebnn-crnn#1]
大佬你好，关于模型结构优化的方面，我这里进行了一些测试，下面是结论：

1、1x1 的 BConv，用来放大或者缩小通道数
2、kxk 的 nn.Conv，在小通道上使用，能保留更多的信息
3、BN，这个层可以不使用，改为softmax，节省更多的空间
4、模型改成沙漏结构，1x1 BConv减小通道数，kxk 的nn.Conv负责在小通道的时候提取卷积信息，1x1 BConv放大通道数，Softmax在通道维度数映射为概率密度。这样构成一个基础的沙漏结构（大通道=Bconv=>小通道=nn.Conv=>小通道=BConv+Softmax=>大通道）

另外：
是否可以添加好友位，你可以注明身份并任意邮件到：hliu1997ac@163.com，我会回复你我的详细信息。

**Comment by Garliccurry** [comment]
您好，已查收
